doc/wiki/move-patient-data-between-facilities.md

```diff
@@ -0,0 +1,18 @@
+# Moving patient data from one facility to another
```

Comment:
There was another doc or section in another doc about this right? Should we link to it? Or deprecate that?

Reply:
That one is for moving a user's data, we'll need them both.